backend/data/static_ward_data.csv

```diff
@@ -1,25 +1,25 @@
-state,district,ward,population
-"MH","Mumbai","A",210847
-"MH","Mumbai","B",140633
-"MH","Mumbai","C",202922
-"MH","Mumbai","D",382841
-"MH","Mumbai","E",440335
-"MH","Mumbai","FS",396122
-"MH","Mumbai","FN",524393
-"MH","Mumbai","GS",457931
-"MH","Mumbai","GN",582007
-"MH","Mumbai","HE",580835
-"MH","Mumbai","HW",337391
-"MH","Mumbai","KE",810002
-"MH","Mumbai","KW",70068
-"MH","Mumbai","L",778218
-"MH","Mumbai","ME",67485
-"MH","Mumbai","MW",41404
-"MH","Mumbai","N",619556
-"MH","Mumbai","PS",437849
-"MH","Mumbai","PN",796775
-"MH","Mumbai","RS",589886
-"MH","Mumbai","RC",513077
-"MH","Mumbai","RN",363827
-"MH","Mumbai","S",691227
-"MH","Mumbai","T",330195
\ No newline at end of file
+state,district,ward,population,Latitude,Longitude,,,,
+MH,Mumbai,A,210847,18.931933,72.827144,,,,
+MH,Mumbai,B,140633,18.956602,72.839636,,,,
+MH,Mumbai,C,202922,18.951744,72.82879,,,,
+MH,Mumbai,D,382841,18.96775,72.810913,,,,
+MH,Mumbai,E,440335,18.97531,72.835832,,,,
+MH,Mumbai,FS,396122,18.9986,72.84693,,,,
+MH,Mumbai,FN,524393,19.02638,72.865506,,,,
+MH,Mumbai,GS,457931,18.99864,72.822582,,,,
+MH,Mumbai,GN,582007,19.03861,72.845914,,,,
+MH,Mumbai,HE,580835,19.068014,72.855329,,,,
+MH,Mumbai,HW,337391,19.065497,72.831932,,,,
+MH,Mumbai,KE,810002,19.10756,72.86446,,,,
+MH,Mumbai,KW,70068,19.109718,72.834215,,,,
+MH,Mumbai,L,778218,19.088508,72.885004,,,,
+MH,Mumbai,ME,67485,19.055609,72.923142,,,,
+MH,Mumbai,MW,41404,19.055968,72.8969,,,,
+MH,Mumbai,N,619556,19.09192,72.92989,,,,
+MH,Mumbai,PS,437849,19.156353,72.859609,,,,
+MH,Mumbai,PN,796775,19.166775,72.79988,,,,
+MH,Mumbai,RS,589886,19.204522,72.86264,,,,
+MH,Mumbai,RC,513077,19.22697,72.853319,,,,
+MH,Mumbai,RN,363827,19.2487,72.8637,,,,
+MH,Mumbai,S,691227,19.137305,72.939501,,,,
+MH,Mumbai,T,330195,19.164618,72.966893,,,,
```

Comment:
Probably done using excel? 
Think you can do one more edit and drop the 4 or so empty columns at the end? those without column names?
And the string qualifiers (the " around it) dropped... 
thats ok for now, but keep that in mind if there is ever a district/ward with a , in its name

Reply:
Added the quotes (didn't think about that corner case but with unsanitized data it's definitely possible) and removed the columns.